core/src/main/scala/org/genivi/sota/core/rvi/SotaServices.scala

```diff
@@ -89,17 +89,15 @@ object SotaServices {
   def register(baseUri: Uri)
               (implicit transport: Json => Future[Json], ec : ExecutionContext) : Future[ServerServices] = {
     val startRegistration = registerService("/sota/start", baseUri.withPath( baseUri.path / "start"))
-    val cancelRegistration = registerService("/sota/cancel", baseUri.withPath( baseUri.path / "cancel"))
```

Comment:
true, it expects this service, though it isn't used anywhere afaik. So I can just remove it from the client.